docs/src/pages/components/pagination/PaginationOutlinedRounded.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Pagination from '@material-ui/lab/Pagination';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        marginTop: theme.spacing(2),
+      },
+    },
+  }),
+);
+
+export default function PaginationOutlinedRounded() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Pagination count={10} variant="outlined" shape="rounded" />
+      <Pagination count={10} variant="outlined" shape="rounded" color="primary" />
```

Comment:
For this demo, what do you think of showing the impact on the two supported variants instead of these 4 demos?

```js
<Pagination count={10} shape="rounded" />
<Pagination count={10} shape="rounded" variant="outlined" />
```

Reply:
Yeah, the intention is for non-unique demos to become visual regression tests. For now it just helps to visualise the styling, since that isn't yet set in stone.